hub-chart/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for KubeStellar-Hub.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# Container/chart image versions. These values are set during the chart release process,
+# please do not change them unless you know what your are doing.
+HELM_VERSION: "3.14.0"
+KUBECTL_VERSION: "1.29.3"
+KUBESTELLAR_VERSION: "0.23.0-alpha.2"
+CLUSTERADM_VERSION: "0.8.2"
+OCM_STATUS_ADDON_VERSION: "0.2.0-rc8"
+OCM_TRANSPORT_PLUGIN_VERSION: "0.1.7"
+
+
+# Control controller log verbosity
+# The global VERBOSITY value will be sued for all controller unless a specific controller verbosity override is specified
+VERBOSITY: 2 # default global value for all controllers
```

Comment:
The default for the transport controller is currently 4. (That is not currently something that the transport controller chart exposes, but as soon as it does, changing the default to 2 would be a behavior change --- not a great thing.)

Reply:
yes all controllers have different defaults... can you work it out with @pdettori and @nirrozenbaum 
e.g. kflex has 0 and it is very verbose already

Reply:
also I realized than kflex, being a dependency, will have to be handled differetly